src/compat/api/validation/encoding/cmds/copyTextureToBuffer.spec.ts

```diff
@@ -0,0 +1,47 @@
+export const description = `
+Tests that you can not call copyTextureToBuffer with compressed textures in compat mode.
```

Comment:
nit. Same comment as the other PR, but basically this is duplicated from the only test's description. Maybe we can make these more general for file-wide so that it's not duplicated.

Reply:
done